.github/workflows/build.yml

```diff
@@ -30,7 +30,7 @@ jobs:
     continue-on-error: ${{ matrix.supported }}
     strategy:
       matrix:
-        variant: [aws-k8s-1.21, aws-k8s-1.22, aws-k8s-1.23, aws-k8s-1.24, aws-k8s-1.25, aws-ecs-1]
+        variant: [aws-k8s-1.21, aws-k8s-1.22, aws-k8s-1.23, aws-k8s-1.24, aws-k8s-1.25, aws-k8s-1.26, aws-ecs-1]
```

Comment:
You will pick it up on a rebase, but just watch out for any kind of merge conflict with the changes in variants. The `aws-k8s-1.21` entry is now dropped.